roles/variables/defaults/main.yml

```diff
@@ -1941,10 +1938,10 @@ kafka_connect_health_check_user: "{{ kafka_connect_basic_users.admin.principal }
 ### Password for authenticated Connect Health Check. Set if using customized security like Basic Auth.
 kafka_connect_health_check_password: "{{ kafka_connect_basic_users.admin.password }}"
 
-### User for authenticated ksqlDB Health Check. Set if using customized security like Basic Auth.
+# User for authenticated ksqlDB Health Check. Set if using customized security like Basic Auth.
```

Comment:
```suggestion
### User for authenticated ksqlDB Health Check. Set if using customized security like Basic Auth.
```

Reply:
the default value of this variable was showing that it is getting defined based on oauth_enabled which was then bringing the oauth_enabled to variables.md file. Since we dont want to expose it I add modified this comment to make it not appear in variables.md. But now I realize that this variable should be in variables.md file. So I think we should redefine it to use new auth_mode in its definition